doc/classes/Node.xml

```diff
@@ -292,7 +292,7 @@
 			<description>
 				Finds the first descendant of this node whose [member name] matches [param pattern], returning [code]null[/code] if no match is found. The matching is done against node names, [i]not[/i] their paths, through [method String.match]. As such, it is case-sensitive, [code]"*"[/code] matches zero or more characters, and [code]"?"[/code] matches any single character.
 				If [param recursive] is [code]false[/code], only this node's direct children are checked. Nodes are checked in tree order, so this node's first direct child is checked first, then its own direct children, etc., before moving to the second direct child, and so on. Internal children are also included in the search (see [code]internal[/code] parameter in [method add_child]).
-				If [param owned] is [code]true[/code], only descendants with a valid [member owner] node are checked.
+				If [param owned] is [code]true[/code], only descendants with a valid [member owner] node are checked. Must be [code]false[/code] if searching from [member SceneTree.root] otherwise [code]null[/code] will always be returned because the root node isn't part of any scene file.
```

Comment:
This seems too specific to me. I agree that we should rephrase this paragraph to make it clearer what this parameter does. But I'm not sure we should focus specifically on the root node case.

Reply:
I do see what you mean about it being too specific, but I think it's important to specify root because it's a unique and useful case. Perhaps we can rephrase it as such?

"If owned is true, only descendants with a valid owner node are checked. This would exclude `SceneTree.root`, orphans without parents, and new children who's `owner` was never set."

Reply:
This method and `find_children` are somewhat of a design mistake that may be addressed next time we can.
There's no reason why `owned` should be set to `true` by default, nor why it's finding children owned by _any_ node. Both aspects make this method very misleading.

With that said I don't think this information should be here. The big clarification following the parameter's description is one good way to overwhelm the reader with redundant and specific information.
At most, the tip in this PR should be a new note at the bottom of the description.

Keep in mind that the reader can naturally come to this same conclusion by acknowledging that the `root` node is not owned by any other node. This tidbit could also be lightly mentioned in `root` itself.

Reply:
> I do see what you mean about it being too specific, but I think it's important to specify root because it's a unique and useful case. Perhaps we can rephrase it as such?
> 
> "If owned is true, only descendants with a valid owner node are checked. This would exclude `SceneTree.root`, orphans without parents, and new children whose `owner` was never set."

Updated the new documentation to be less focused on implementation details and to better reflect use cases.